models/schemas/Identifier-Reference.yaml

```diff
@@ -0,0 +1,7 @@
+type: object
+description: A link to a resource representing the person or set of individuals to whom the medication will be given.
+required:
+  - identifier
+properties:
+  identifier:
+    $ref: "Identifier-OdsOrganizationCode.yaml"
```

Comment:
Maybe include OdsOrganizationCode in the filename so nobody accidentally uses it thinking it represents a reference to any identifier?